backend/de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/model/InterfaceWrapperHelper.java

```diff
@@ -1200,6 +1201,15 @@ public static ClientAndOrgId getClientAndOrgId(@NonNull final Object model)
 		return ClientAndOrgId.ofClientAndOrg(clientId, orgId);
 	}
 
+	@NonNull
+	public static UserId getUpdatedBy(@NonNull final Object model)
+	{
+		return getValue(model, "UpdatedBy")
+				.map(userIdObj -> NumberUtils.asInt(userIdObj, -1))
```

Comment:
(only if u have to do other changes too)

consider using de.metas.util.lang.RepoIdAwares#ofObject(java.lang.Object, java.lang.Class<T>, java.util.function.IntFunction<T>)

Reply:
done